client/src/app.jsx

```diff
@@ -2,8 +2,13 @@ import React from 'react';
 // eslint-disable-next-line no-unused-vars
 import ReactDOM from 'react-dom';
 import Main from './components/relatedProducts/Main.jsx';
+<<<<<<< HEAD
 import Overview from './components/overview/overview.jsx';
 import $ from 'jquery';
+=======
+// import $ from 'jquery';
+import RatingsAndReviews from './components/Reviews/Reviews.jsx';
+>>>>>>> 6115ae46967e5ad1adf303c873410a7b63c6625f
```

Comment:
For some reason, the <<<<<Head thingy got left in here. It happened in my file too.

Reply:
Checking